### ISSUE TITLE
Neither Opera nor Opera Mini works with autofill

### ISSUE THREAD

**Issue by mwoz123** [PhilippC/kp2a_accservice_autofill#18]
Auto fill searches DB for incorrect url 
Instead of search for valid url it searches for : "com.opera.browser".

**Comment by Jon999999** [comment]
Have you ever found a solution to this issue? I am seeing the same behaviour lately on Opera, Firefox, Chrome, and FF Focus. I've opened an issue at https://github.com/PhilippC/keepass2android/issues/938 because I don't know whether this is a plugin issue or a main app issue.

**Comment by mwoz123** [comment]
No, I believe it's not implemented or thoose browsers doesn't expose api for this.
